kyuubi-main/pom.xml

```diff
@@ -72,8 +72,18 @@
         </dependency>
 
         <dependency>
-            <groupId>org.apache.hive</groupId>
```

Comment:
let's keep it

Reply:
Restored but left some comments, looks ok to you?

Reply:
To be honest, I prefer using `spark-hive-thriftserver` to retrieve a relative clean hive jdbc driver. But ok to copy-paste all hive stuffs from spark `pom.xml`

Reply:
in main we are isolated with all spark stuffs as they are belong to an engine process，we only need a jdbc client and a server instance

Reply:
one more thing，for server we shall be able to test a hive-jdbc with MVN -D

Reply:
make sense, updated